packages/docs-site/src/components/presenters/footer/footer.scss

```diff
@@ -1,9 +1,15 @@
 @import "@royalnavy/css-framework";
 
 .rn-footer {
+  margin-top: spacing(5);
   background: color(primary, 800);
 }
 
+.rn-footer__container {
```

Comment:
We don't need to specify a container class for the `footer` or a component. This padding can sit on `.rn-footer`.

Reply:
agreed, simplified